lib/algos/list/basic.js

```diff
@@ -1,12 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const Extension = require('./Extension').default;
+
 const checkLimit = require('./tools').checkLimit;
 const DEFAULT_MAX_KEYS = 10000;
 
 /**
  *  Class of an extension doing the simple listing
  */
-class List {
+class List extends Extension {
```

Comment:
I don't think it's an issue. The benchmark compares to the ES5 implementation, so it's 4x slower than the ES5 implementation but fast enough for us since the code would spend nowhere near time number of times making as many super calls as the benchmark.